CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Changed
+- The project Golang version is updated from the end-of-life v1.15 to version v1.16.
```

Comment:
This seems like a change that isn't technically "user facing", and it isn't a security-related change. So I'm thinking that we should leave out this entry in the CHANGELOG.md. Here are our Community guidelines for what should be included in CHANGELOG.md entries:
https://github.com/cyberark/community/blob/main/Conjur/CONTRIBUTING.md#changelog-content

Reply:
Ok, I removed that.